transmart-schemas/src/main/resources/db/changelog/changes/20191907041159_biomart_user_views_with_bitset.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <preConditions onFail="WARN">
+        <dbms type="PostgreSQL" />
+        <sqlCheck expectedResult="1">
+            SELECT count(*) FROM pg_available_extensions WHERE name='pg_bitcount'
+        </sqlCheck>
+    </preConditions>
+    <changeSet author="artur@thehyve.nl" id="20191907041159-0">
+        <comment>
+            create biomart_user schema
```

Comment:
This comment looks a bit like the following :smiley:
```java
// Assigns the value 3 to x
x = 3
```

Reply:
yep... removed :-)